pybaseball/utils.py

```diff
@@ -90,6 +91,35 @@ def date_range(start: date, stop: date, step: int, verbose: bool = True) -> Iter
         low += timedelta(days=step)
 
 
+def statcast_date_range(start: date, stop: date, step: int, verbose: bool = True) -> Iterator[Tuple[date, date]]:
+    '''
+    Iterate over dates. Skip the offseason dates. Returns a pair of dates for beginning and end of each segment.
+    Range is inclusive of the stop date.
+    If verbose is enabled, it will print a message if it skips offseason dates.
+    This version is Statcast specific, relying on skipping predefined dates from STATCAST_VALID_DATES.
+    '''
+    low = start
+
+    while low <= stop:
+        year_start, year_end = STATCAST_VALID_DATES.get(low.year,
```

Comment:
suggest naming this `season_start`, `season_end` instead of `year_start`, `year_end`